vars.yaml

```diff
@@ -187,6 +187,10 @@ vars:
         SENTRY_CLIENT_ENVIRONMENT: '{instance}'
         SENTRY_CLIENT_RELEASE: '{sentry_release}'
         SENTRY_TAG_SERVICE: wsgi
+        C2C_SECRET: adminc2c
+        LOG_VIEW_ENABLED: TRUE
+        SQL_PROFILER_ENABLED: TRUE
+        DEBUG_VIEW_ENABLED: TRUE
```

Comment:
I did a mistake in the c2cwsgiutils README. They should all be prefixed with `C2C_`.

If you have redis, you can even add that:
```
C2C_REDIS_URL: redis://redis:6379
C2C_BROADCAST_PREFIX: broadcast_geoportal_
```

And add all the C2C_* variables in tilecloudchain as well

Reply:
C2C_REDIS_URL and C2C_BROADCAST_PREFIX are already present in the CONST_vars.yaml

Reply:
Fixed, thanks :-)